courses-app/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,14 @@
+import {NgModule} from "@angular/core";
+import {CourseComponent} from "../features/course/course.component";
+
+
+@NgModule({
+  imports: [],
+  declarations: [
+    CourseComponent
+  ],
+  exports: [
+    CourseComponent
+  ],
```

Comment:
You should import and export FontAwesome here. Also declare and export info, button and header